R/auto_tweet.R

```diff
@@ -107,17 +146,23 @@ action_auto_tweet <- function(twitter_token = ser_token, google_drive_auth = dri
   if (retweet_day) {
     retweet_queue <- post_tweet_of_type(retweet_queue)
   } else {
-    tweet_queue <- post_tweet_of_type(tweet_queue)
+    tweet_library_status <- post_tweet_of_type
+  }
```

Comment:
It looks like you changed the code such that `post_tweet_of_type` is not a function but the result of a post. The first part of this logic treats it as a function still, and the second part creates an object you need later, `tweet_library_status`. If it's a retweet day, this will fail, right? Even if you change the first error, `post_retweet()` doesn't return anything called `just_tweeted`, so it will fail later.